### ISSUE TITLE
Py2exe does not build a working executable

### ISSUE THREAD

**Issue by Menelion** [Oire/TheQube#65]
Something weird started happening with this upcoming release: on both of my machines (Windows 7 and Windows 10) `Py2exe` does not build a working executable: the build process goes flawlessly, but the resulting exe file runs and closes immediately.  
@quartizer is able to build a working file, but after InnoSetup script assembles a bundle, it is 184 megs heavy (mine is 11.3 megs only).  
@craigbrett17 or someone else, any thoughts?

**Comment by craigbrett17** [comment]
@Oire: I'm not able to set up successfully either. I can setup install fine, but when I then try to setup py2exe, I run into issues, similar to the ones here: http://stackoverflow.com/questions/34505408/py2exe-fails-to-create-exe-due-to-missing-dll-when-opencv-is-imported

I believe I need some DLLs here when I run the installer that the various things we use expect to be there when they are compiled. Maybe because I'm on Win 10, the versions are newer. 

I tried implementing the second answer to that thread, with the big long list of DLLExcludes, but it wasn't enough, and I carried on playing whack a mole for a while. But I'm worried by doing this I'll make a bad installer that doesn't work.
